pkg/clusters/cleanup.go

```diff
@@ -84,7 +84,12 @@ func (c *Cleaner) Cleanup(ctx context.Context) error {
 			namespaceClient := c.cluster.Client().CoreV1().Namespaces()
 
 			if err := namespaceClient.Delete(ctx, namespace.Name, metav1.DeleteOptions{}); err != nil {
-				if !errors.IsNotFound(err) {
+				switch errors.IsNotFound(err) {
+				// If the namespace cannot be found then we're good to go.
+				case true:
+					return nil
+				// If there was a different error then report it to the caller.
+				case false:
 					return err
```

Comment:
```suggestion
				if errors.IsNotFound(err) {
					// If the namespace cannot be found then we're good to go.
					return nil
				}
				return err
```

Let's get back to `if` 😅 . WDYT?